app/javascript/lib/study-overview/terra-profile-warning.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export const profileWarning =
+  <div className="container">
+    <p className="bg-danger">
+      The visualization above is attempting to fetch data from this study's private bucket, but it cannot issue an
+      access token for the request as you have not completed your Terra profile. This is required in order to set your
+      access permissions on the bucket. Please&nbsp;
+      <a href="https://support.terra.bio/hc/en-us/articles/360028235911-How-to-register-for-a-Terra-account"
```

Comment:
At demo, Tim or Amelia suggested less depth in these messages, which I agree would help.  Perhaps:

```suggestion
      This visualization is trying to fetch data from the study's private bucket, but it cannot as you have not completed your Terra profile. Please&nbsp;
      <a href="https://support.terra.bio/hc/en-us/articles/360028235911-How-to-register-for-a-Terra-account"
```